service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantAboutToStartControllerTest.java

```diff
@@ -26,8 +48,53 @@ void shouldPrepopulateApplicantDataWhenNoApplicantExists() {
             .data(Map.of("data", "some data"))
             .build();
 
+        given(authTokenGenerator.generate()).willReturn(serviceAuthToken);
+        given(organisationApi.findOrganisationById(userAuthToken, serviceAuthToken))
+            .willReturn(Organisation.builder().build());
+
         AboutToStartOrSubmitCallbackResponse callbackResponse = postAboutToStartEvent(caseDetails);
 
         assertThat(callbackResponse.getData()).containsKey("applicants");
     }
+
+    @BeforeEach
```

Comment:
let's put the beforeEach above all the tests, then you can remove the duplicated mocks from the above test

Reply:
done :)